src/omnicore/rpctx.cpp

```diff
@@ -551,12 +552,13 @@ Value omni_sendsto(const Array& params, bool fHelp)
     uint32_t propertyId = ParsePropertyId(params[1]);
     int64_t amount = ParseAmount(params[2], isPropertyDivisible(propertyId));
     std::string redeemAddress = (params.size() > 3 && !ParseText(params[3]).empty()) ? ParseAddress(params[3]): "";
+    uint32_t distributionPropertyId = (params.size() > 4) ? ParsePropertyId(params[4]) : 0;
```

Comment:
Passing a ```0``` to ```CreatePayload_SendToOwners``` (which is the default if not supplied) is how we let it know that it should create a v0, rather than a v1 STO.

Since v1 STOs are 100x more expensive (due to their ability to cross properties) I figured it would be a bit rough to start sending *all* STOs as v1, so unless a distribution property is supplied a v0 STO is created.